etl/src/main/java/org/pih/hivmigration/etl/sql/ReturnVisitDateMigrator.groovy

```diff
@@ -0,0 +1,65 @@
+package org.pih.hivmigration.etl.sql
+
+class ReturnVisitDateMigrator extends SqlMigrator{
```

Comment:
Per above, can we rename this to something more generic that will allow us to add more Obs to this in the future?